src/components/CurrencyLogo/CurrencyLogoComponent.stories.mdx

```diff
@@ -0,0 +1,29 @@
+import { DAI, WETH } from '@swapr/sdk'
+
+import { Canvas, Meta, Story, ArgsTable } from '@storybook/addon-docs'
+import { CurrencyLogoComponent } from './CurrencyLogo.component'
+
+import { NATIVE_CURRENCY_LOGO } from './CurrencyLogo.utils'
+
+<Meta title="Currencies" component={CurrencyLogoComponent} />
+
+# CurrencyLogoComponent
+
+Show the currency logo.
+
+<Canvas>
+  <Story
+    name="CurrencyLogoComponent"
+    argTypes={{
+      sources: {
+        control: 'select',
+        options: [[NATIVE_CURRENCY_LOGO[1]], [NATIVE_CURRENCY_LOGO[100]]],
```

Comment:
The option name look weird, is there a way to make it better?

Reply:
It looks but I didn't find a way to give it a label. Will ship like this and improve it in the future.